post_all_articles.py

```
@@ -0,0 +1,56 @@
+    post_data = {
+        "title": title,
+        "content": content,
+        "status": "draft"
+    }
+    
+    print(f"\nSending article: {title}")
+    print(f"Content length: {len(content)} characters")
+    
+    # Send to Zapier
+    try:
+        response = requests.post(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)